test/http_downloads_test.rb

```diff
@@ -8,7 +8,7 @@ def command(src, dst)
 
 class HttpDownloadsTest < Test::Unit::TestCase
   def test_should_download_a_file
-    assert Proxy::HttpDownloads.start_download 'source', 'destination'
+    refute Proxy::HttpDownloads.start_download('source', 'destination').nil?
```

Comment:
use `assert_not_nil` instead. Better still, as @domcleal suggested verify that it returns an instance of expected class..